_sass/_hero.scss

```diff
@@ -0,0 +1,28 @@
+.hero {
+  align-items: center;
+  background: $ember-orange;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+  width: 100vw;
+}
+
+.hero__code {
+  background: $navy;
+  border-radius: 4px;
+  padding: 1.25rem 3rem;
+
+  code {
+    font-size: 1.25rem;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.